requirements.txt

```diff
@@ -40,3 +40,4 @@ vobject==0.6.6
 wsgiref==0.1.2
 XlsxWriter==0.7.7
 xlwt==0.7.5
+openupgradelib>=1.1.0
```

Comment:
This is the configuration for doing it: https://github.com/OCA/openupgradelib/blob/master/.travis.yml#L26-L32